src/rsz/src/Resizer.cc

```diff
@@ -3648,4 +3648,79 @@ void Resizer::resetInputSlews()
   input_slew_map_.clear();
 }
 
+void Resizer::eliminateDeadLogic()
+{
+  std::vector<const Instance*> queue;
```

Comment:
std::queue ?

Reply:
Needlessly complicated to me, we don't need FIFO here